acpreprocessing/stitching_modules/convert_to_n5/acquisition_dir_to_ngff.py

```diff
@@ -34,9 +42,21 @@ def get_pixel_resolution_from_rootdir(
     with md_path.open() as f:
         md = json.load(f)
     xy = md["settings"]["pixel_spacing_um"]
-    z = md["positions"][1]["x_step_um"]
+    z = md["positions"][0]["x_step_um"]
     return [xy, xy, z]
 
+def get_strip_positions_from_rootdir(
+        root_dir, md_bn="acqinfo_metadata.json"):
+    root_path = pathlib.Path(root_dir)
+    md_path = root_path / md_bn
+    with md_path.open() as f:
+        md = json.load(f)
+    # FIXME how to do proper version comparison?
+    if md["version"] == "0.0.3":
```

Comment:
We have used `packaging.version.parse` other places in this repo.  see example here:
https://github.com/AllenInstitute/axonal_connectomics/blob/e20be7b1d8306ef25477bafaa67d433fde1076b0/acpreprocessing/utils/ac_imageio.py#L19-L24